service/src/main/kotlin/io/provenance/digitalcurrency/consortium/stream/EventStreamConsumer.kt

```diff
@@ -103,6 +128,8 @@ class EventStreamConsumer(
 
     fun String.uniqueHash(index: Int): String = "$this-$index"
 
+    // private fun io.provenance.attribute.v1.Attribute.bankUuid(): UUID = this.value.toByteArray().toUuid()
```

Comment:
does this need to be here for later?

Reply:
I added the code back that pulls nycb tags via the chain vs. the db call. It's commented out, but it's useful for local testing because we don't have the same database data that Jim has.